prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-presubmits.yaml

```diff
@@ -0,0 +1,149 @@
+presubmits:
+  GoogleContainerTools/kpt-config-sync:
+  - name: kpt-config-sync-presubmit
+    cluster: build-kpt-config-sync
+#    branches: ["master"]
```

Comment:
Is this intentional?

Reply:
This is an artifact copied over from the reference job config, where it is also commented out

Reply:
The branch filter was commented out so that the presubmit job can be triggered on any branches.